lib/ledger_sync/adaptors/quickbooks_online/account/ledger_serializer.rb

```diff
@@ -5,15 +5,28 @@ module Adaptors
     module QuickBooksOnline
       module Account
         class LedgerSerializer < QuickBooksOnline::LedgerSerializer
-          attribute ledger_attribute: 'Id', resource_attribute: :ledger_id
-          attribute ledger_attribute: 'Name', resource_attribute: :name
-          attribute ledger_attribute: 'AccountType', resource_attribute: :account_type, type: LedgerSerializerType::AccountType
-          attribute ledger_attribute: 'AccountSubType', resource_attribute: :account_sub_type, type: LedgerSerializerType::AccountSubType
-          attribute ledger_attribute: 'AcctNum', resource_attribute: :number
-          attribute ledger_attribute: 'CurrencyRef.value', resource_attribute: :currency
-          attribute ledger_attribute: 'Classification', resource_attribute: :classification, type: LedgerSerializerType::ClassificationType
-          attribute ledger_attribute: 'Description', resource_attribute: :description
-          attribute ledger_attribute: 'Active', resource_attribute: :active
+          attribute ledger_attribute: 'Id',
+                    resource_attribute: :ledger_id
+          attribute ledger_attribute: 'Name',
+                    resource_attribute: :name
+          attribute ledger_attribute: 'AccountType',
+                    resource_attribute: :account_type,
+                    type: LedgerSerializerType::AccountType
+          attribute ledger_attribute: 'AccountSubType',
+                    resource_attribute: :account_sub_type,
+                    type: LedgerSerializerType::AccountSubType
+          attribute ledger_attribute: 'AcctNum',
+                    resource_attribute: :number
+          attribute ledger_attribute: :CurrencyRef,
```

Comment:
If currency is a resource, shouldnt this be a relationship with `ledger_id: ‘USD’`?

Reply:
It is handled in the type.

Reply:
I know, but its a relationship. Seems like we're introducing yet another another pattern how we set value. I think if we take away the dogma that ID is a numeric value, we can easily justify `LedgerSync::Currency.new(ledger_id: 'USD', name: 'United States Dollar')`.

And even if some ledger will use numeric ID for a currency instead of `USD`, it will be already done. Coz we will just do `LedgerSync::Currency.new(ledger_id: 123, name: 'United States Dollar')`

Reply:
Ok, to be fair, saying for QBO that currency has `ledger_id: 'USD'` when almost all other resources has numeric value is introducing complexity. Tho classes has weird IDs.

Reply:
I'm not sure I agree with the ID stuff, though to be fair, I don't think we can assume it's numeric.  I think we most always convert to a string anyway.  

That said, I do agree that using the type was adding duplicate ways of achieving something.  I just switched it to using a currency serializer (similar to how we do with other `references_many` relationships).

That work?